### ISSUE TITLE
Set up issues - git clone and creating a folder structure on your machine you can live with. 

### ISSUE THREAD

**Issue by hifilorau** [Code-the-Dream-School/CtD-Week-1-Git-Assignment#1]
Originally this was a broken link. Now it is an example of where to potentially put questions worth talking about as a group.

**Comment by Hvierma20** [comment]
Hi, the youtube video doesn't open, it says private.

**Comment by hifilorau** [comment]
@Hvierma20 Think we should be good to go now, let me know if it isn't. Thanks for catching that.

**Comment by Hvierma20** [comment]
I can see the video, my batch isn't working :( I'm going to ask in the mentor session.

Thanks

**Comment by hifilorau** [comment]
@Hvierma20 Did you install git yet? That is most likely the case. Let me know what kind of machine, etc you have. It should be in the treehouse material but happy to post here as it may be helpful to other students. (if we go that route though, i'd going to make a new issue so that it's easier to find). Just let me know.

**Comment by Hvierma20** [comment]
@hifilorau yes I installed it, I'm using a thinkpad lenovo. here is a screen shot of my git bash, I committed the assignment (Or I think I did it) but haven't pushed it.   
[06.02.2020_github.docx](https://github.com/Code-the-Dream-School/CtD-Week-1-Git-Assignment/files/4719787/06.02.2020_github.docx)

**Comment by hifilorau** [comment]
@Hvierma20 - there is a place in the shell where you ran -

`git add repo-location.git`

where you should have run

`git clone you-repo-location`

Try that from an empty folder in your bash and that should download the repository into a new folder on your computer. But before you do that you might want to make a 'lessons' or 'code' folder on your users/hvierma so it's users/hvierman/code or /lessons and then clone the repository there.
